packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -1173,18 +1181,23 @@ class _InteractiveViewerBuilt extends StatelessWidget {
   final bool constrained;
   final Matrix4 matrix;
   final Alignment? alignment;
+  final bool transformChild;
 
   @override
   Widget build(BuildContext context) {
-    Widget child = Transform(
-      transform: matrix,
-      alignment: alignment,
-      child: KeyedSubtree(
-        key: childKey,
-        child: this.child,
-      ),
+   Widget child = KeyedSubtree(
+      key: childKey,
+      child: this.child,
     );
 
+    if (transformChild) {
```

Comment:
Don't forget to add a test that verifies that a `Transform` widget was properly added (or not!) when `transformChild` is used.

Reply:
Will do!